network/src/connectivity_manager/mod.rs

```diff
@@ -595,11 +594,9 @@ where
             // to generate the new eligible peers set.
             let new_eligible = self.peer_pubkeys.union_all();
 
-            let peer_pubkeys = &self.peer_pubkeys;
-            info!(
-                "{} current pubkeys: update src: {}, all peer pubkeys: {}, new eligible set: {:?}",
-                self.network_context, src, peer_pubkeys, new_eligible,
-            );
+            send_struct_log!(network_log("EligiblePeersUpdate", &self.network_context)
+                .field(DISCOVERY_SOURCE, &src)
```

Comment:
i'm guessing we're using `.field` when we use a key in multiple places?

Reply:
Yeah, it was the thought that you could make a typed field, so that you don't accidentally put the wrong type in.